html.go

```diff
@@ -65,22 +75,14 @@ func (h *HTML) Load(names ...string) error {
 
 	for _, n := range names {
 		filename := fmt.Sprintf("html-templates/%s.tpl", n)
-		b, err := loadWithOverride(h.overridePath, filename)
-		if err != nil {
-			return err
-		}
-
-		clone, err := h.layout.Clone()
-		if err != nil {
-			return err
-		}
 
-		_, err = clone.New("authboss").Funcs(h.funcMap).Parse(string(b))
+		FSPath := filepath.Join(h.overridePath, filename)
+		tpl, err := template.New(path.Base(FSPath)).Funcs(h.funcMap).ParseFS(h.templatesFS, FSPath)
 		if err != nil {
-			return errors.Wrapf(err, "failed to load template for page %s", n)
+			return errors.Wrapf(err, "failed to load embedded template for page %s", n)
```

Comment:
@yusufmalikul Maybe let's keep the original string, because in the normal situation, this may not be `embedded` file?

Reply:
that makes sense.

Reply:
@yusufmalikul thanks